phpmd.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" ?>
+<ruleset
+  name="Web Stories PHPMD rule set"
+  xmlns="http://pmd.sf.net/ruleset/1.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://pmd.sf.net/ruleset/1.0.0
+                       http://pmd.sf.net/ruleset_xml_schema.xsd"
+  xsi:noNamespaceSchemaLocation="
+                       http://pmd.sf.net/ruleset_xml_schema.xsd"
+>
+
+  <rule ref="rulesets/codesize.xml">
+    <exclude name="CyclomaticComplexity" />
+  </rule>
+  <rule ref="rulesets/codesize.xml/CyclomaticComplexity">
+    <properties>
+      <property name="reportLevel" value="20" />
+    </properties>
+  </rule>
+
+  <rule ref="rulesets/cleancode.xml">
+    <exclude name="ElseExpression" />
+    <exclude name="StaticAccess" />
+  </rule>
+
+  <rule ref="rulesets/unusedcode.xml">
+    <exclude name="UnusedFormalParameter" />
+  </rule>
+
```

Comment:
I think we should add `BooleanArgumentFlag` to exclude list. WordPress uses boolean flag params on methods and functions all over the place. I personally don't agree with this rule.

Reply:
Undecided 🤷 

It was only flagged in a few places, where I suppressed the warnings. That should suffice for now.